FabricObserver/Observers/AppObserver.cs

```diff
@@ -1784,6 +1878,40 @@ private Task<ParallelLoopResult> MonitorDeployedAppsAsync(CancellationToken toke
                         checkMemPct = true;
                     }
 
+                    // Memory - Private Bytes MB.
+                    if (application.ErrorPrivateBytesMb > 0 || application.WarningPrivateBytesMb > 0 || MonitorResourceGovernanceLimits)
+                    {
+                        _ = AllAppPrivateBytesDataMb.TryAdd(id, new FabricResourceUsageData<float>(ErrorWarningProperty.PrivateBytesMb, id, 1, false, EnableConcurrentMonitoring));
+                    }
+
+                    if (AllAppPrivateBytesDataMb.ContainsKey(id))
+                    {
+                        checkMemPrivateBytes = true;
+                    }
+
+                    // Memory - RG monitoring.
+                    if (MonitorResourceGovernanceLimits && repOrInst.RGEnabled && repOrInst.RGMemoryLimitMb > 0)
+                    {
+                        _ = AllAppRGMemoryUsagePercent.TryAdd(id, new FabricResourceUsageData<double>(ErrorWarningProperty.RGMemoryUsagePercent, id, 1, false, EnableConcurrentMonitoring));
+                    }
+
+                    if (AllAppRGMemoryUsagePercent.ContainsKey(id))
+                    {
+                        rgMemoryPercentThreshold = application.WarningRGMemoryLimitPercent;
+
+                        if (rgMemoryPercentThreshold > 0)
+                        {
+                            if (rgMemoryPercentThreshold < 1)
+                            {
+                                rgMemoryPercentThreshold = application.WarningRGMemoryLimitPercent / 100; // decimal to double.
+                            }
+                        }
+                        else
+                        {
+                            rgMemoryPercentThreshold = 90; // Default: 90%.
```

Comment:
do we need to specify 0.9 here, since above we are using application.WarningRGMemoryLimitPercent / 100 ?

Reply:
oh, I've missed the part that we are dividing by 100 only in case rgMemoryPercentThreshold is below 1.

Reply:
so, in case rgMemoryPercentThreshold is below 1, do we need to multiply it by 100, instead of dividing it - to have the same logic used further?

Reply:
Yes. That is a bug.